src/v3/bandwidthRtc.ts

```diff
@@ -8,7 +8,6 @@ import { AudioLevelChangeHandler, BandwidthRtcError, RtcAuthParams, RtcOptions,
 import { PublishMetadata, PublishSdpAnswer, PublishedStream, StreamMetadata, SubscribeSdpOffer, CodecPreferences } from "./types";
 import Signaling from "./signaling";
 import AudioLevelDetector from "../audioLevelDetector";
-import DtmfSender from "../dtmfSender";
```

Comment:
I say delete this file as well, but no big deal either way

Reply:
That's kind of what I was mentioning earlier, `v2` still relies on it.

Reply:
Right, I see now... Thanks for being patient with me :)